docs/src/docs/asciidoc/user/management.adoc

```diff
@@ -102,32 +102,40 @@ This way, all managed objects get registered into a common `ManagementRegistry`
 
 == Rules for statistics calculation
 
-This table describes the impact of each cache method on the statistics. It is compatible with
+This table describes if yes or no, a cache method can impact a given statistic. It is compatible with
```

Comment:
This describes whether or not a cache method can impact a given statistic. It is compatible with